app/admin/general_info_items.rb

```diff
@@ -0,0 +1,13 @@
+ActiveAdmin.register GeneralInfoItem do
+  permit_params :name, :content
+
+  index do
+    selectable_column
+    id_column
+    column :name
+    column :content
```

Comment:
maybe the content column will have too much data. What do you think about transforming the content text and just show a preview here. You could use the decorator pattern (with the [draper gem](https://github.com/drapergem/draper)).
The decorator could have a method called `preview` which could be something like:
```ruby
def preview
  content.truncate(30)
end
```